packages/geo/src/lib/map/shared/map.ts

```diff
@@ -175,7 +175,8 @@ export class IgoMap {
 
   updateView(options: MapViewOptions) {
     const currentView = this.ol.getView();
-    const viewOptions = Object.assign(
+    // to update the zoom we need to delete 'resolution' property
+    const {resolution, ...viewOptions} = Object.assign(
       {
         zoom: currentView.getZoom()
       },
```

Comment:
La résolution est une valeur valider pour les options, il ne faudrait pas complètement l'ignorer. Je crois qu'on veut privilégier le zoom si la résolution n'est pas défini dans les options.

* Le `zoom: currentView.getZoom()` ne semble pas servir a rien?
Voici une suggestion:
```suggestion
    const newOptions: MapViewOptions = {
      ...currentView.getProperties(),
      ...options
    };

    if (options.zoom && options.resolution == null) {
      newOptions.resolution = undefined;
    }

    this.setView(newOptions);
```